src/SamplesApp/SamplesApp.UITests/_Utils/ColorCodeParser.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Drawing;
+using System.Linq;
+
+namespace SamplesApp.UITests._Utils
+{
+	internal static class ColorCodeParser
+	{
+		public static Color Parse(string colorCode)
+		{
+			if (colorCode == null)
+			{
+				throw new ArgumentNullException(nameof(colorCode));
+			}
+
+			if (!colorCode.StartsWith("#"))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this call to 'colorCode.StartsWith' to an overload that accepts a 'StringComparison' as a parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4919453)